server/src/pages/shot/editor.js

```diff
@@ -590,6 +590,14 @@ exports.Editor = class Editor extends React.Component {
     }
   }
 
+  isClickOnPicker(e) {
+    const targetClass = e.target.className;
+    if (targetClass === "swatch" || targetClass === "color-board") {
```

Comment:
You could simply return the result of this or expression instead of using an if.